src/ui/wxWidgets/I18N/pos/pwsafe_de.po

```diff
@@ -5956,3739 +5956,33 @@ msgstr "Deutsch"
-#~ "\t\thas differences in the following fields:"
-#~ msgstr ""
-#~ "\tEintrag - Gruppe:\"%s\"; Titel:\"%s\"; Benutzer:\"%s\"\n"
-#~ "\t\that Unterschiede in folgende Felder:"
-
-#~ msgid "\tGroup:'%s'; Title:'%s'; User:'%s'"
-#~ msgstr "\tGruppe:'%s'; Titel:'%s'; Benutzer:'%s'"
-
-#~ msgid "\tNo entries were found containing search string '%s'"
-#~ msgstr "\tKeine Einträge enthalten den Suchtext '%s'"
-
-#~ msgid "\tThe following entries contain search string '%s'"
-#~ msgstr "\tFolgende Einträge enthalten den Suchtext '%s'"
-
-#~ msgid ""
-#~ "\tThe following restrictions were selected:\n"
-#~ "\t\t %s"
-#~ msgstr ""
-#~ "\tDie folgenden Einschränkungen wurden ausgewählt:\n"
-#~ "\t\t %s"
-
-#~ msgid "an unnamed file"
-#~ msgstr "eine unbenannte Datei"
-
-#~ msgid "and"
-#~ msgstr "und"
-
-#~ msgid "big endian"
-#~ msgstr "Big Endian"
-
-#~ msgid "bottom"
-#~ msgstr "Ende"
-
-#~ msgid "day(s)"
-#~ msgstr "Tag(e)"
-
-#~ msgid "email"
-#~ msgstr "Email"
-
-#~ msgid "in"
-#~ msgstr "in"
-
-#~ msgid "little endian"
-#~ msgstr "Little Endian"
-
-#~ msgid ""
-#~ "multiple entries exist with title=%s and username=%s or with group=%s and "
-#~ "title=%s."
-#~ msgstr ""
-#~ "Mehrere Einträge mit Titel %s und Benutzername %s oder Gruppe %s und "
-#~ "Titel %s existieren."
-
-#~ msgid "multiple entries exist with title=%s."
-#~ msgstr "Mehrere Einträge mit Titel %s existieren."
-
-#~ msgid "no entry exists in the root with title=%s and username=%s."
-#~ msgstr ""
-#~ "Kein Eintrag mit Titel %s und Benutzername %s besteht in der Wurzel."
-
-#~ msgid "no entry exists with group=%s, title=%s and empty username."
-#~ msgstr ""
-#~ "Kein Eintrag mit Gruppe %s, Titel %s und leerem Benutzernamen besteht."
-
-#~ msgid "no entry exists with group=%s, title=%s and username=%s."
-#~ msgstr "Kein Eintrag mit Gruppe %s, Titel %s und Benutzername %s besteht."
-
-#~ msgid ""
-#~ "no entry exists with title=%s and username=%s or with group=%s, title=%s."
-#~ msgstr ""
-#~ "Kein Eintrag mit Titel %s und Benutzername %s oder mit Gruppe %s und "
-#~ "Titel %s besteht."
-
-#~ msgid "no entry exists with title=%s."
-#~ msgstr "Kein Eintrag mit Titel=%s besteht."
-
-#~ msgid "pixels"
-#~ msgstr "Pixels"
-
-#~ msgid "shortcut to %s"
-#~ msgstr "Verknüpfung mit %s"
-
-#~ msgid "static"
-#~ msgstr "statisch"
-
-#~ msgid "subset: '%s' field '%s' '%s' %s"
-#~ msgstr "Teilmenge: '%s' Feld '%s' '%s' %s"
+msgid "Bring All to Front"
+msgstr "Alle Einblenden"
 
-#~ msgid "this entry's password and data"
-#~ msgstr "Passwort und Daten dieses Eintrags"
+msgid "Quit PasswordSafe (wxWidgets)"
+msgstr "PasswordSafe beenden"
+msgid "Hide PasswordSafe (wxWidgets)" msgstr "PasswordSafe ausbelenden"
 
-#~ msgid "top"
-#~ msgstr "Beginn"
+# wxString standard strings
+msgid "Create new document"
+msgstr "Erzeuge ein neues Dokument"
 
-#~ msgid "whatever"
-#~ msgstr "egal"
+# wxString standard strings
+msgid "Open an existing document"
+msgstr "Öffnen eines vorhandenen Dokumentes"
```

Comment:
Just a suggestion.

- "Ein vorhandenes Dokument öffnen"

Reply:
Hmm, should the translation of "document" possibly be replaced by "database" in the context of pwsafe?
However, I am not a Mac user and so I am not familiar with the concept of a Mac user interface, nor with the standard menu.

Reply:
To not deviate too much from English translation I prefer to stay with document/Dokument.